saq/queue/postgres.py

```diff
@@ -132,10 +138,50 @@ async def init_db(self) -> None:
             if result and not result[0]:
                 return
 
-            for statement in DDL_STATEMENTS:
+            migrations = return_migrations(
+                migrations_table=self.migrations_table,
+                jobs_table=self.jobs_table,
+                stats_table=self.stats_table,
+            )
+            target_number = len(migrations)
```

Comment:
`target_version`?

Reply:
don't use len(...), let's use the file number